### PR TITLE
Add example of local pipeline testing to README.md

### PR DESCRIPTION
**Pull request by tomjw64**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,124 @@
...
+    ports: ["8080:8080"]
+    # ...
+
+  docker-registry:
+    restart: always
+    image: registry:2
+    ports: ["5000:5000"]
+
+  git-server:
+    image: tomjw64/git-server
```
Comment (philschatz) [inline README.md]: Could this be transferred over to openstax?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,124 @@
...
+$ git push local-pipeline --all
+$ fly -t dev unpause-pipeline -p my-stuff-image
+$ git tag my-tag
+$ git push local-pipeline my-tag
+# ...We wait for our build to succeed and then check our insecure docker registry for the image
+$ curl http://localhost:5000/v2/my-stuff/tags/list
+# {"name":"my-stuff","tags":["latest"]}
+```
+
+Hooray! Our pipeline did it's job. Now we have to modify the pipeline back to point to the remote services:
```
Comment (philschatz) [inline README.md]:
```suggestion
Hooray! Our pipeline did its job. Now we have to modify the pipeline back to point to the remote services:
```

File: README.md [hunk 3]
```diff
@@ -2,6 +2,124 @@
...
+```yml
+# pipeline.yml
+resources:
+  - name: source-code-tagged
+    type: git
+    source:
+      uri: http://git-server:8000/my-stuff  # Change this line to point to remote repo
```
Comment (philschatz) [inline README.md]:
```suggestion
      uri: http://git-server:8000/my-stuff  # Change this line to point to remote repo. https://github.com/openstax/my-repo
```

Just to be super-clear to people reading this

File: README.md [hunk 4]
```diff
@@ -2,6 +2,124 @@
...
+      tag_filter: '*'
+
+  - name: docker-hub-image
+    type: docker-image
+    source:
+      repository: docker-registry:5000/my-stuff  # Change this line to point to remote registry
```
Comment (philschatz) [inline README.md]: What is the public URL for the docker hub registry?

File: README.md (hunk 1 above)
Comment (tomjw64) [reply README.md]: I can if people want to use it. I didn't want to litter the openstax org with stuff nobody uses.

Comment (philschatz) [reply README.md]: I see your point... if we're suggesting that people use it for local testing of concourse then it seems like it'd be something that others _would_ use.

**Pull request merged**